### PR TITLE
Finished the homework 5

### PR DESCRIPTION
**Pull request by MaximIurceac**
Please check the HW

### CODE REVIEW

File: src/main/java/org/example/BonusTask/JavaClock.java
Comment (gh3rtzz) [inline src/main/java/org/example/BonusTask/JavaClock.java:1]: This file should be removed. Nothing is happening inside of it

**Review by gh3rtzz — CHANGES_REQUESTED** [review]
Remove the unnecessary file. otherwise - approved

**Review by gh3rtzz — CHANGES_REQUESTED** [review]
file still not removed

**Comment by MaximIurceac** [comment]
I did it

**Review by gh3rtzz — APPROVED** [review]
approved

**Pull request merged**